packages/relay-runtime/network/RelayObservable.js

```diff
@@ -454,6 +463,17 @@ function isObservable(obj) {
   );
 }
 
+// Use declarations to teach Flow how to check isAsyncIterable.
+declare function isAsyncIterator(p: mixed): boolean %checks(p instanceof
+  AsyncIterator);
+
+function isAsyncIterator(obj) {
+  return typeof val < 'u' &&
+    val !== null &&
+      (val[Symbol.toStringTag] === 'AsyncGenerator' ||
+        (Symbol.asyncIterator && Symbol.asyncIterator in val));
+}
```

Comment:
shouldn't this be checking `obj`, not `val`?

Reply:
🤦🏻 ya... i had it as obj, but copied the body from elsewhere when I realized the Symbol isnt a thing on Safari. Have fixed up.